### PR TITLE
MakeGaudiMap: use target properties to get the necessary command and …

### PR DESCRIPTION
**Pull request by andresailer**
…library locations

BEGINRELEASENOTES
- MakeGaudiMap: better inference of listcomponents and library location by using target properties
- MakeGauiMap: Detailed information about the command is not only printed when VERBOSE=1

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
Still have to run on macos, but gitlab is currently down

@MarkusFrankATcernch Can you try with this as well?

**Comment by andresailer** [comment]
Works: https://gitlab.cern.ch/sailer/DD4hep/pipelines/1430325

File: cmake/MakeGaudiMap.cmake [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
   #  Running listcomponents from DD4hep
-  #   
-  FIND_PROGRAM( Gaudi_listcomponents_CMD listcomponents PATHS ${genmap_install_dir}/../bin ${DD4hep_DIR}/bin
-    NO_DEFAULT_PATH NO_CMAKE_PATH)
-  GET_FILENAME_COMPONENT(GAUDI_LISTCOMP_INSTALL ${Gaudi_listcomponents_CMD} DIRECTORY)
-  ###MESSAGE( STATUS " *** MakeRootMap ${genmap_install_dir}/../bin $ENV{DD4hep_DIR}/bin ${DD4hep_DIR}" )
-  MESSAGE( STATUS " *** MakeGaudiMap.cmake run command : ${Gaudi_listcomponents_CMD} -o ${rootmapfile} ${libname}
-                 WORKING_DIRECTORY ${genmap_install_dir} "
+  #
+  MESSAGE( STATUS " *** MakeGaudiMap.cmake run command : ${DD4HEP_LISTCOMPONENTS_CMD} -o ${rootmapfile} ${libname}
```
Comment (petricm) [inline cmake/MakeGaudiMap.cmake]: It might be better to convert these two status messages to `dd4hep_debug`?

Comment (andresailer) [reply cmake/MakeGaudiMap.cmake]: This runs in an independent cmake run.
Thus there is no dd4hep_debug

**Pull request merged**